packages/gallery/src/components/item/videos/videoItem.js

```diff
@@ -280,13 +279,16 @@ class VideoItem extends GalleryComponent {
           utils.deviceHasMemoryIssues() ||
           this.state.ready ||
           !shouldCreateVideoPlaceholder(this.props.styleParams)
-            ? { backgroundColor: 'black' }
+            ? {
```

Comment:
There's too much code in the JSX, I prefer to avoid ocmplicated JSX. There are two things I would do here:

1. Reuse the duplicated `imageDimensions` (make it your start value that you change based on the condition
2. Move the entire style calculation to a function (not inside the JSX) 
3. If `this.state.ready` || `utils.deviceHasMemoryIssues()` - we don't want any styles at all? Is that the case?

Reply:
1,2 - OK, sounds better
3 - in that case we don't want the background image